TOC.md

```diff
@@ -2,184 +2,205 @@
-    - [Transaction Model](sql/transaction-model.md)
-    - [Isolation Levels](sql/transaction-isolation.md)
-  - [Constraints](sql/constraints.md)
-  - [Generated Columns](sql/generated-columns.md)
-  - [Connectors and APIs](sql/connection-and-APIs.md)
-  - [Error Codes and Troubleshooting](sql/error.md)
-  - [Compatibility with MySQL](sql/mysql-compatibility.md)
-  - [TiDB Memory Control](sql/tidb-memory-control.md)
-  - [Slow Query Log](sql/slow-query.md)
-  + Advanced Usage
-    - [Read Data From History Versions](op-guide/history-read.md)
-    - [Garbage Collection (GC)](op-guide/gc.md)
-+ TiDB Operations Guide
-  - [Hardware and Software Requirements](op-guide/recommendation.md)
-  + Deploy
-    - [Ansible Deployment (Recommended)](op-guide/ansible-deployment.md)
-    - [Offline Deployment Using Ansible](op-guide/offline-ansible-deployment.md)
-    - [Binary Deployment](op-guide/binary-deployment.md)
-    - [Docker Deployment](op-guide/docker-deployment.md)
-    - [Docker Compose Deployment](op-guide/docker-compose.md)
-    - [Cross-DC Deployment Solutions](op-guide/cross-dc-deployment.md)
-    - [Kubernetes Deployment](op-guide/kubernetes.md)
-  + Configure
-    + By Component
-      - [tidb-server](op-guide/configuration.md)
-      - [pd-server](op-guide/pd-configuration.md)
-      - [tikv-server](op-guide/tikv-configuration.md)      
-    - [Configuration File Description](op-guide/tidb-config-file.md)
-    - [Modify Component Configuration Using Ansible](op-guide/ansible-deployment-rolling-update.md#modify-component-configuration)
-    - [Enable TLS Authentication](op-guide/security.md)
-    - [Generate Self-signed Certificates](op-guide/generate-self-signed-certificates.md)
-    - [Cluster Topology Configuration](op-guide/location-awareness.md)
-  + Monitor
-    - [Monitoring Framework Overview](op-guide/monitor-overview.md)
-    - [Monitor a TiDB Cluster](op-guide/monitor.md)
-    + Key Monitoring Metrics
-      - [Overview](op-guide/dashboard-overview-info.md)
-      - [TiDB](op-guide/tidb-dashboard-info.md)
-      - [PD](op-guide/dashboard-pd-info.md)
-      - [TiKV](op-guide/dashboard-tikv-info.md)
-  + Scale
-    - [Scale a TiDB Cluster](op-guide/horizontal-scale.md)
-    - [Scale Using Ansible](op-guide/ansible-deployment-scale.md)
-  + Upgrade
-    - [Upgrade the Component Version](op-guide/ansible-deployment-rolling-update.md#upgrade-the-component-version)
-    - [TiDB 2.0 Upgrade Guide](op-guide/tidb-v2.0-upgrade-guide.md)
-    - [TiDB 2.1 Upgrade Guide](op-guide/tidb-v2.1-upgrade-guide.md)
-    - [FAQs After Upgrade](op-guide/upgrade-faq.md)
-  - [Tune Performance](op-guide/tune-tikv.md)
-  + Backup and Migrate
+      + Transport Layer Security (TLS)
+        - [Enable TLS For MySQL Clients](sql/encrypted-connections.md)
+        - [Enable TLS Between TiDB Components](op-guide/security.md)
+      - [Generate Self-signed Certificates](op-guide/generate-self-signed-certificates.md)
+    - Monitoring and Alerting
```

Comment:
On further thought, can we move Monitoring and Security sections up by a level (to be the same as Deploy/Migrate), so that we can differ with CRDB? This way Deploy will also be less stuffed than it is now.